src/store/sagas/positions.ts

```diff
@@ -179,13 +179,23 @@ function* handleInitPosition(action: PayloadAction<InitPositionData>): Generator
     closeSnackbar(loaderSigningTx)
     yield put(snackbarsActions.remove(loaderSigningTx))
 
-    yield put(
-      snackbarsActions.add({
-        message: 'Failed to send. Please try again.',
-        variant: 'error',
-        persist: false
-      })
-    )
+    if (e.message === 'Cancelled') {
```

Comment:
We shouldn't assume that every wallet will return an error with 'cancelled.' We need to consider a general failure when failed in signing. In this PR I got the same message in snackbar as in master:
![image](https://github.com/invariant-labs/webapp-a0/assets/43304803/6490a9bd-aa33-4f1c-be3f-d2770941c248)

Reply:
helpful would be to add hook to ensure error. It could be something like this :

```
export const ensureError = (value: unknown): Error => {
  if (value instanceof Error) return value;

  let stringified = "[Unable to stringify the thrown value]";

  stringified = JSON.stringify(value);

  const error = new Error(
    `This value was thrown as is, not through an Error: ${stringified}`
  );
  return error;
};
```
```
    const ensuredError = ensureError(error)

    if(ensuredError.message === 'Cancelled'){   }
      
```